easybuild/easyconfigs/b/Bowtie2/Bowtie2-2.4.4-GCC-10.3.0.eb

```diff
@@ -21,9 +23,22 @@ description = """ Bowtie 2 is an ultrafast and memory-efficient tool for alignin
 toolchain = {'name': 'GCC', 'version': '10.3.0'}
 toolchainopts = {'pic': True, 'cstd': 'gnu++98'}
 
-source_urls = [('https://sourceforge.net/projects/bowtie-bio/files/%(namelower)s/%(version)s', 'download')]
-sources = ['%(namelower)s-%(version)s-source.zip']
-checksums = ['b9fc1d6e5b898f832c590023bcb15e2a7b76e0ffac486800a9897e7cba2175fc']
+source_urls = [
+    ('https://sourceforge.net/projects/bowtie-bio/files/%(namelower)s/%(version)s', 'download'),
+    'https://github.com/simd-everywhere/simde-no-tests/archive/refs/heads',
```

Comment:
@SIB-software please use a specific commit instead of just the tip of the master branch.

for example, this is the url for the latest commit (as of now):
https://github.com/simd-everywhere/simde-no-tests/archive/cbef1c1.tar.gz

you should also rename the file to a separate version for simd-everywhere, corresponding to the commit date, e.g.:
```
_simd_everywhere_version = '20220504'
```

Reply:
This is what I tried at the beginning but it fails: the `_simde_version` variable does not look to be interpreted in `sources`. This eb file contains also modifications recommended by @boegel 
[Bowtie2-2.4.4-GCC-10.3.0.eb.txt](https://github.com/easybuilders/easybuild-easyconfigs/files/10269097/Bowtie2-2.4.4-GCC-10.3.0.eb.txt)

```
== FAILED: Installation ended unsuccessfully (build directory: /home/vitsoft/eb_builddir/build/Bowtie2/2.4.4/GCC-10.3.0): build failed (first 300 chars): Couldn't find file 
simde-%(_simde_version)s.zip anywhere, and downloading it didn't work either... Paths attempted (in order): 
/var/vhosts/vitsoft-home/sib-easyconfigs.git/easybuild/easyconfigs/b/Bowtie2/b/Bowtie2/simde-%(_simde_version)s.zip, /var/vhosts/vitsoft-home/sib-easyconfigs.git/easybuild/e (took 0 secs)
== Results of the build can be found in the log file(s) /tmp/eb-an0_19g1/easybuild-Bowtie2-2.4.4-20221220.145827.FqMRm.log
Fetching files: 100% (2/2) ━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━ 0:00:00 (simde-%(_simde_version)s.zip)
ERROR: Build of /var/vhosts/vitsoft-home/sib-easyconfigs.git/easybuild/easyconfigs/b/Bowtie2/Bowtie2-2.4.4-GCC-10.3.0.eb failed (err: "build failed (first 300 chars): Couldn't find file simde-%(_simde_version)s.zip anywhere, and downloading it didn't work either... Paths attempted (in order): /var/vhosts/vitsoft-home/sib-easyconfigs.git/easybuild/easyconfigs/b/Bowtie2/b/Bowtie2/simde-%(_simde_version)s.zip, /var/vhosts/vitsoft-home/sib-easyconfigs.git/easybuild/e")
```

Reply:
done

Reply:
> So instead of something like `"%(_simde_version)s.tar.gz"`, use `"%s.tar.gz" % _simde_version` instead

yes, that's what I meant, sorry for not explaining this.

Reply:
@boegel I think I know what happened here, you wanted to tag @smoretti instead of @smoors, right?